.travis.yml

```diff
@@ -4,6 +4,8 @@ python:
 - '3.7'
 services:
 - postgresql
+# For Celery (web2py scheduler replacement).
+- redis-server
```

Comment:
interesting.... Once this is in place we can move to using redis to manage session data instead of Postgres.

Reply:
Hmmm - interesting. But I don't see any way to do that -- session data gets stored to either a file or a DB table, and DAL doesn't support redis AFAIK. Or am I missing something? (I hope so -- that sounds really neat...)

Reply:
http://web2py.com/books/default/chapter/29/13?search=cache#Sessions-in-Redis

Reply:
web2py also supports using redis in places where you would also use memcache.

Reply:
Wow, cool! Nice find!